docs/pages/patterns/multi-input.md

```diff
@@ -586,7 +594,8 @@ To add text in the `body` of the component, simply include your text in the `fd-
         </div>
     </div>
     <div class="fd-popover__body fd-popover__body--no-arrow fd-popover__body--dropdown fd-popover__body--dropdown-fill" aria-hidden="true" id="F4GcKJH8a">
-        <ul class="fd-list fd-list--multi-input fd-list--has-message">
+      <div class="fd-popover__body__no-overflow">
```

Comment:
This cannot work. Something called `fd-popover__body--something`  should be applied on the `fd-popover__body` element.

Reply:
this was the wrapper which we had discussed about